integration-tests/actions/vrf/common/actions.go

```diff
@@ -306,26 +315,29 @@ func CreateVRFKeyOnVRFNode(vrfNode *VRFNode, l zerolog.Logger) (*client.VRFKey,
 	return vrfKey, pubKeyCompressed, nil
 }
 
-func FundNodesIfNeeded(ctx context.Context, existingEnvConfig *vrf_common_config.ExistingEnvConfig, client blockchain.EVMClient, l zerolog.Logger) error {
+func FundNodesIfNeeded(existingEnvConfig *vrf_common_config.ExistingEnvConfig, client *seth.Client, l zerolog.Logger) error {
 	if *existingEnvConfig.NodeSendingKeyFundingMin > 0 {
 		for _, sendingKey := range existingEnvConfig.NodeSendingKeys {
 			address := common.HexToAddress(sendingKey)
-			sendingKeyBalance, err := client.BalanceAt(ctx, address)
+			sendingKeyBalance, err := client.Client.BalanceAt(context.Background(), address, nil)
```

Comment:
why it was changed back to use `context.Background`?

Reply:
will fix